packages/prerender/src/graphql/graphql.ts

```diff
@@ -0,0 +1,97 @@
+import fs from 'fs'
+import path from 'path'
+import { DocumentNode, print } from 'graphql'
+
+import { getPaths } from '@redwoodjs/internal'
+import { getOperationName } from '@redwoodjs/web'
+
+export async function executeQuery(
+  gqlHandler: (args: any) => Promise<any>,
+  query: DocumentNode,
+  variables?: Record<string, unknown>
+) {
+  const operationName = getOperationName(query)
+  const operation = { operationName, query: print(query), variables }
+  const handlerResult = await gqlHandler(operation)
+
+  return handlerResult.body
+}
+
+export async function getGqlHandler() {
+  const gqlPathTs = path.join(getPaths().api.functions, 'graphql.ts')
```

Comment:
I don't think you need to specify the path here. I think - but am not 100% sure - import and require are smart enough to handle extension names automatically

Reply:
Ohh, that'd be nice. I'll try!

Reply:
Looks like you were right. I've now simplified this code and it seems to still be working 🤞